client/src/features/visualizer-threejs/constants.ts

```diff
@@ -22,19 +23,16 @@ export const ANIMATION_TIME_SECONDS = 3;
 
 // colors
 export const PENDING_BLOCK_COLOR = new Color("#A6C3FC");
-
-export const ACCEPTED_BLOCK_COLORS = [new Color("#0101FF"), new Color("#0000DB"), new Color("#0101AB")];
-
-export const COLORS = [
-    new Color("#F0F4FF"),
-    new Color("#E0EAFF"),
-    new Color("#C8DAFE"),
-    PENDING_BLOCK_COLOR,
-    new Color("#82A5F8"),
-    new Color("#5C84FA"),
-    new Color("#2559F5"),
-    ...ACCEPTED_BLOCK_COLORS,
-];
+export const ACCEPTED_BLOCK_COLOR = new Color("#0101AB");
+export const CONFIRMED_BLOCK_COLOR = new Color("#0000DB");
+export const FINALIZED_BLOCK_COLOR = new Color("#0101FF");
+// TODO Remove accepted state once is added to the SDK (missing)
+export const BLOCK_STATE_TO_COLOR = new Map<BlockState | "accepted", Color>([
+    ["pending", PENDING_BLOCK_COLOR],
```

Comment:
couldnt these colors be written as `BlockState.Pending` so we avoid magic strings? 
```suggestion
    [BlockState.Pending, PENDING_BLOCK_COLOR],
```

Reply:
Unfortunately not cause it's not an ENUM, but a string union type.
It is type-safe though, if you try to use a value that is not part of `BlockState` TS will complain.

Reply:
Yeah, seems like doing an enum would be redundant